src/components/Topbar/Topbar.jsx

```diff
@@ -0,0 +1,78 @@
+import { useEffect, useState } from "react";
+import { IoLogoTux } from "react-icons/io";
+import { Button } from "../Button/Button";
+import styles from "./Topbar.module.scss";
+import { IconContext } from "react-icons";
+import { Link, useNavigate } from "react-router-dom";
+import { routes } from "../../navigation/routes";
+import { useStore } from "../../context/store";
+import { Pill } from "../Pill/Pill";
+
+const navLinks = [
+  { name: "Home", path: routes.HOME },
+  { name: "Services", path: routes.SERVICES },
+  { name: "About Us", path: routes.ABOUT_US },
+];
+
+export function Topbar() {
+  const [menuOpen, setMenuOpen] = useState(false);
+
+  const navigate = useNavigate();
+  const user = useStore((state) => state.user);
+  const initializeUser = useStore((state) => state.initializeUser);
+  const logOutUser = useStore((state) => state.logOutUser);
+
+  useEffect(() => {
```

Comment:
šis  useEffect turėtų būti atliktas App faile, arba naujai sukurtame <Initialize /> komponente, 
užkrauti user'į tikrai nėra TopBar atsakomybė